lib/order.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'text'
+
+class Order
+
+  attr_accessor :menu, :text
+
+  def initialize(text = Text_message.new)
+    @menu = @menu = {
+      "Sushi set" => 12,
+      "Prawn katsu curry" => 9,
+      "Kitsune udon" => 7,
+      "Onigiri" => 3,
+      "Matcha ice-cream" => 4
+    }
+    @total = 0
+    @order = []
+    @text = text
+  end
+
+  def print_menu
+    @menu.each { |key, value| puts " #{key}: £#{value}" }
+  end
+
+  def user_order(food)
+    if @menu.include?(food) == true
+      @order.push(food)
+      print @order
+    elsif food == 'done'
```

Comment:
It looks like this method is responsible mainly for adding an item of food to the order 

This branch of the if statement looks like a separate piece of logic responsible for printing the order total and sending the text. We can use this separation of concerns as a great reason to put this logic into a different method.

The way I try to think about this (when it comes to both methods and classes):
- identify the main purpose of the method/class
- check that each part of the code is only implementing that main purpose, and if it isn't move that logic into another method/class